R/table.R

```diff
@@ -63,31 +63,49 @@ html_table.xml_node <- function(x, header = NA, trim = TRUE,
 
   ncols <- lapply(cells, html_attr, "colspan", default = "1")
   ncols <- lapply(ncols, as.integer)
+  nrows <- lapply(cells, html_attr, "rowspan", default = "1")
+  nrows <- lapply(nrows, as.integer)
+
   p <- unique(vapply(ncols, sum, integer(1)))
+  maxp <- max(p)
 
-  if (length(p) > 1) {
+  if (length(p) > 1 & (maxp * n != Reduce(sum, lapply(nrows, sum)))) {
```

Comment:
I don't follow the logic here. Could you refactor to be a bit clearer? (Is that reduce equivalent to `sum(unlist(nrows))`?